src/app/state/movies/movies.facade.ts

```diff
@@ -0,0 +1,18 @@
+import { Movie } from 'src/app/interfaces';
+import { Injectable } from '@angular/core';
+import { Store } from '@ngrx/store';
+import { AppState } from './../index';
+import * as  fromMovie from './movies.actions';
+
+@Injectable()
+export class MovieFacade {
+  public movies$ = this.store;
```

Comment:
no to jest źle, co prawda na razie tego nie mialeś robić, więc od bidy to zostaw i poprawisz w następnym korku. Ale usodtępniasz tutaj na fasadzie property movies$ które linkuje do całego store, bez sensu